dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -221,7 +221,7 @@
     /**
      * The url of the reference service
      */
-    protected final transient List<URL> urls = new ArrayList<URL>();
+    protected final transient List<URL> urls = new ArrayList<>();
```

Comment:
**Code Structure Issue**: Diamond operator ("<>") should be used for instantiation instead of specifying the type on both sides. This makes the code cleaner and easier to read. <br> **Fix**: Use the diamond operator for instantiation to improve code readability. <br> **Code Suggestion**: 
 ```
 -    protected final transient List<URL> urls = new ArrayList<URL>();
 +    protected final transient List<URL> urls = new ArrayList<>();
 ```